src/Indexer.js

```diff
@@ -69,6 +69,20 @@ export default class Indexer {
     })
   }
 
+  // https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-asciifolding-tokenfilter.html
+  indexSettings() {
+    return {
+      analysis : {
+        analyzer : {
+          default : {
+            tokenizer : 'standard',
+            filter : ['asciifolding']
+          }
+        }
+      }
+    }
+  }
```

Comment:
i had considered using the second example here:  https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-asciifolding-tokenfilter.html

for something like the following:
```javascript
  indexSettings() {
    return {
      analysis : {
        analyzer : {
          default : {
            tokenizer : 'standard',
            filter : ['my_ascii_folding']
          }
        },
        filter : {
          my_ascii_folding : {
            type : 'asciifolding',
            preserve_original : true
          }
        }
      }
    }
  }
```

because those docs said:
> Accepts `preserve_original` setting which defaults to false but if true will keep the original token as well as emit the folded token.

but as the tests show, the more basic setup from the first example in that doc still allows searching on the original (diacritic containing) phrase.  happy to use the second example (and to add a test showing the difference it makes) if someone more knowledgeable in elasticsearch thinks there's good reason to.

FWIW, i just tried, and the tests in this PR still pass with the more detailed setting.  🤷‍♂